cdc/redo/reader/file.go

```diff
@@ -361,7 +361,9 @@ func shouldOpen(startTs, endTs uint64, name, fixedType string) (bool, error) {
 	if filepath.Ext(name) == common.TmpEXT {
 		return true, nil
 	}
-	return commitTs > startTs && commitTs <= endTs, nil
+	// the commitTs=max(ts of log item in the file), if max > startTs then should open,
+	// filter out ts in (startTs, endTs] for consume
+	return commitTs > startTs, nil
```

Comment:
Does this mean we could loss data when applying redo logs before this PR?

Reply:
yes